diff --git a/sandbox/pyodide/setup.sh b/sandbox/pyodide/setup.sh
index 7c7d55e5..f0cc7945 100755
--- a/sandbox/pyodide/setup.sh
+++ b/sandbox/pyodide/setup.sh
@@ -8,8 +8,10 @@ echo "## Get pyodide node package"
 
 if [[ ! -e _build/worker ]]; then
   mkdir -p _build/worker
-  cd _build/worker
-  yarn init --yes
-  yarn add pyodide@0.23.4
-  cd ../..
+  pushd _build/worker
+    yarn init --yes
+    touch yarn.lock
+    echo 'nodeLinker: "node-modules"' > ".yarnrc.yml"
+    yarn add pyodide@0.23.4
+  popd
 fi
